docs/how-to/use-trained-models.txt

```diff
@@ -115,42 +115,46 @@ selecting checkpoints.
 
    most_recent_checkpoint = trial.select_checkpoint(latest=True)
 
-   specific_checkpoint = trial.select_checkpoint(uuid="uuid-for-checkpoint")
+   specific_checkpoint = client.get_checkpoint(uuid="uuid-for-checkpoint")
 
 ****************************
  Using the Checkpoint Class
 ****************************
 
-The :class:`~determined.experimental.Checkpoint` class can both download
-the checkpoint from persistent storage and load it into memory in a
-Python process.
+The :class:`~determined.experimental.client.Checkpoint` class can both
+download the checkpoint from persistent storage and load it into memory
+in a Python process.
 
-The :meth:`~determined.experimental.Checkpoint.download` method
+The :meth:`~determined.experimental.client.Checkpoint.download` method
 downloads a checkpoint from persistent storage to a directory on the
 local file system. By default, checkpoints are downloaded to
 ``checkpoints/<checkpoint-uuid>/`` (relative to the current working
-directory). The :meth:`~determined.experimental.Checkpoint.download`
-method accepts ``path`` as an optional parameter, which changes the
-checkpoint download location.
+directory). The
+:meth:`~determined.experimental.client.Checkpoint.download` method
+accepts ``path`` as an optional parameter, which changes the checkpoint
+download location.
 
 .. code:: python
 
-   from determined.experimental import Determined
+   from determined.experimental import client
 
-   checkpoint = Determined().get_experiment(id).top_checkpoint()
+   client.login()
+   checkpoint = client.get_experiment(id).top_checkpoint()
    checkpoint_path = checkpoint.download()
 
    specific_path = checkpoint.download(path="specific-checkpoint-path")
 
-The :meth:`~determined.experimental.Checkpoint.load` method downloads
-the checkpoint, if it does not already exist locally, and then loads it
-into memory in a Python process, as shown in the following snippet.
+The :meth:`~determined.experimental.client.Checkpoint.load` method
+downloads the checkpoint, if it does not already exist locally, and then
+loads it into memory in a Python process, as shown in the following
+snippet.
 
 .. code:: python
 
-   from determined.experimental import Determined
+   from determined.experimental import client
 
-   checkpoint = Determined().get_experiment(id).top_checkpoint()
+   client.login()
```

Comment:
can we remove these `client.login()` lines?  They're often not necessary, and when they are necessary it's unlikely that  an empty `login()` call will work.

A better strategy is to point to the overall `client` docs right at the top of the document and let that be enough.